python/lsst/daf/butler/formatters/fitsExposureFormatter.py

```diff
@@ -109,11 +89,19 @@ def readInfoComponent(self, component):
         obj : component-dependent
             In-memory component object.
```

Comment:
Should this method raise an exception if the requested component is not understood? Rather than returning None?

Reply:
Now raising KeyError.